rollup.config.js

```diff
@@ -0,0 +1,56 @@
+import resolve from "rollup-plugin-node-resolve";
+import commonjs from "rollup-plugin-commonjs";
+import babel from "rollup-plugin-babel";
+import { uglify } from "rollup-plugin-uglify";
+import license from "rollup-plugin-license";
+import { stripIndent } from 'common-tags';
+
+const uglifyOptions = {
+  mangle: false,
+  compress: false,
+  output: {
+    beautify: true,
+    indent_level: 2
+  }
+}
+
+export default {
+  input: "app/javascript/active_admin/base.js",
+  output: {
+    file: "app/assets/javascripts/active_admin/base.js",
+    format: "umd",
+    name: "ActiveAdmin"
+  },
+  plugins: [
+    resolve(),
+    commonjs(),
+    babel(),
+    uglify(uglifyOptions),
+    license({
+      banner: {
+        commentStyle: 'none',
+        // Add rails-style imports for sprockets usage
+        content:stripIndent`
+          //= require jquery3
+          //= require jquery-ui/widgets/datepicker
+          //= require jquery-ui/widgets/dialog
+          //= require jquery-ui/widgets/sortable
+          //= require jquery-ui/widgets/tabs
+          //= require jquery-ui/widget
+          //= require jquery_ujs
+          //= require_self
+        `.trim()
```

Comment:
Using a license header plugin for this job is really smart, hehehe :+1:. I don't think we need the `trim()` call though? Also, maybe we add `+ "\n"` instead to make the output a little bit prettier (I'm used to a blank line after requires)?